flink-runtime/src/main/java/org/apache/flink/runtime/dispatcher/Dispatcher.java

```diff
@@ -792,7 +792,11 @@ public CompletableFuture<Acknowledge> disposeSavepoint(String savepointPath, Tim
 
                     try {
                         Checkpoints.disposeSavepoint(
-                                savepointPath, configuration, classLoader, log);
+                                savepointPath,
+                                new Configuration(),
```

Comment:
Why using an empty job configuration here?

Reply:
Because disposeSavepoint is a cluster-wide operation, so it does not require job-specific configuration.